pkg/healthcheck/healthcheck.go

```diff
@@ -136,6 +136,10 @@ const (
 	/// plugin is installed and ready
 	LinkerdCNIPluginChecks CategoryID = "linkerd-cni-plugin"
 
+	// LinkerdOpaquePortsDefinitionChecks adds checks to validate that the opaque-ports
+	// label has been defined both in the service and the corresponding pods
```

Comment:
Sorry I'm being so annoying with these comments but can we please wrap them at 80 col? 😅

Reply:
Sure!